docs/modules/third_party_rules_callbacks.md

```diff
@@ -265,6 +265,24 @@ server_.
 
 If multiple modules implement this callback, Synapse runs them all in order.
 
+### `on_threepid_unbind`
+
+_First introduced in Synapse v1.63.0_
+
+```python
+async def on_threepid_unbind(
+    user_id: str, medium: str, address: str, identity_server: str
+) -> Tuple[bool, bool]:
+```
+
+Called before a threepid association is removed.
+
+It should return a tuple of 2 booleans reporting if a changed happened for the first, and if unbind
+needs to stop there for the second (True value). In this case no other module unbind will be
+called, and the default unbind made to the IS that was used on bind will also be skipped.
+In any case the mapping will be removed from the Synapse 3pid remote table, except if an Exception
+was raised at some point.
```

Comment:
I have a few issues with this documentation:

* it doesn't explain what the arguments passed to the callback mean
* it doesn't mention that the module can then do its own custom unbinding (and should probably hint at using it together with `on_threepid_bind`)
* it's quite unclear what the return values mean:
  * the first boolean reports if a change happens, but what is a change in this context?
  * the second boolean indicates whether unbind needs to stop there, but what happens if it's `False
* it doesn't mention what happens when there are multiple callbacks registered

Feel free to have a look at the documentation for other callbacks for inspiration.